app/src/main/res/values/strings.xml

```diff
@@ -1570,7 +1570,7 @@ Partially means that a wheelchair can enter and use the restroom, but no handrai
     <string name="bicycle_boulevard">Bicycle boulevard</string>
     <string name="separate_cycleway_no">No cycling</string>
     <string name="separate_cycleway_allowed">Cyclists are admitted</string>
-    <string name="separate_cycleway_no_or_allowed">Not designated for cyclists (regardless of whether they are admitted or prohibited) </string>
+    <string name="separate_cycleway_no_or_allowed">Not designated for cyclists (regardless of whether cycling is admitted or prohibited) </string>
```

Comment:
maybe "permitted" or "allowed"? "admitted" seems less clear

Reply:
For the string above there too then?

Reply:
"Admit" sounds more like "allowed as a guest", which is why I chose that word

Reply:
> For the string above there too then?

Yes

Reply:
I believe "allowed" is what you want.  "Admitted" is a formal way to say "allowed entry into" and would typically be used in cases where someone is showing a ticket for admission.